core/src/test/scala/org/apache/spark/util/collection/SorterSuite.scala

```diff
@@ -241,6 +360,13 @@ class IntArraySortDataFormat extends AbstractIntArraySortDataFormat[Int] {
   }
 }
 
+class ByteArraySortDataFormat extends AbstractByteArraySortDataFormat[Byte] {
```

Comment:
I forget it I asked -- why do we need this abstract parent class? can these be collapsed?

Reply:
This is I also tried but again the test became pretty slow.

Reply:
Also surprised by that; I can't think of a way this would be faster. I guess this is how the code it is copying is constructed, so OK.